examples/tcp-wait/server.rs

```diff
@@ -23,6 +23,9 @@ use ::std::{
     },
     net::SocketAddr,
 };
+use demikernel::runtime::network::socket::option::SocketOption;
```

Comment:
If `SocketOption` will be exposed to user-applications, could you re-export it at the `libs.rs` level?